### PR TITLE
Fix stream and worker names, use suffixes

### PR DESCRIPTION
**Pull request by simao**
Add integration test for common-messaging

### CODE REVIEW

File: common-messaging/src/test/resources/reference.conf [hunk 1]
```diff
@@ -0,0 +1,12 @@
+messaging {
+  kinesis {
+    appName = "sota-common-messaging"
+    regionName = "eu-central-1"
+    streamNameSuffix = "test"
```
Comment (CalumMcCall) [inline common-messaging/src/test/resources/reference.conf]: Can you make a ticket to change the '-dev' suffix streams on aws to '-test'?

Comment (simao) [inline common-messaging/src/test/resources/reference.conf]: but this is just to run this test, not for dev. I created this stream.

Comment (CalumMcCall) [inline common-messaging/src/test/resources/reference.conf]: Yeah, I just realized, it must exist for the test to pass. My bad.

**Comment by CalumMcCall** [comment]
10/11

**Pull request merged**